drivers/Kconfig

```diff
@@ -89,4 +89,6 @@ source "drivers/neural_net/Kconfig"
 
 source "drivers/hwinfo/Kconfig"
 
+source "drivers/lora/Kconfig"
```

Comment:
Can you move the lora Kconfig setting after the `drivers/ieee802154/Kconfig` line so that we have various radio technologies near each other.

Reply:
Okay